src/Components/Topics.jsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+const Topics = ({ topics }) => {
+
 
-const Topics = () => {
     return (
         <>
-            <h2>Topics</h2>
+            <div className="topic-btn-container">
+            <h3>Topics</h3>
+                {topics.map((topic) => {
+                    return (
+                        <button className='topic-select-btn' ><Link to={`/${topic.slug}`} >{topic.slug.charAt(0).toUpperCase() + topic.slug.slice(1)}</Link></button>
```

Comment:
Wrapping the whole button in the Link might be better here.
I was clicking the button and it wasn't working because the Link is only around the text you have to make sure you're clicking exactly the text and not even a little outside of its 'hit box'